src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -303,6 +306,17 @@ public RowMap getRow() throws Exception {
 		}
 	}
 
+	private SSLMode getBinlogSSLMode(String sslMode) {
+		if (sslMode != null) {
+			for (SSLMode mode : SSLMode.values()) {
+				if (mode.toString().equals(sslMode)) {
+					return mode;
+				}
+			}
+		}
+		return SSLMode.DISABLED;
```

Comment:
should add a warning string here that the user passed an invalid mode string.

Reply:
done!